pipelines/gitops-pull-request-rhtap/gitops-pull-request.yaml

```diff
@@ -30,11 +30,11 @@ spec:
     - description: The Rekor host that EC should use to look up transparency logs
       name: ec-rekor-host
       type: string
-      default: http://rekor-server.rhtap.svc
+      default: http://rekor-server.rhtap-tas.svc
     - description: The TUF mirror that EC should use
       name: ec-tuf-mirror
       type: string
-      default: http://tuf.rhtap.svc
+      default: http://tuf.rhtap-tpa.svc
```

Comment:
The TUF mirror is in the `tpa` namespace? Isn't it also part of TAS?

Reply:
Good catch! Fixed.